frontend/src/modules/controllers/ValueController.tsx

```diff
@@ -12,6 +16,21 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+/**
+ * @typedef Props
+ *
+ * Props interface for the value controller
+ *
+ * @prop {number} value Paramater value displayed in UI
+ * @prop {string} label Label to associate to which value belongs to
+ * @prop {function} onClick Callback with value on button click or Rotary encoder
+ * @prop {number} min Minimum Range of an Alarm (Default to 0)
+ * @prop {number} max Minimum Range of an Alarm (Default to 100)
```

Comment:
If `ValueController` is for Parameters values, why are min and max described as being for alarms?

Reply:
Yes that was a typo error. It is not specific to parameter, It has min/max because it also uses `ValueClicker` component